runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -63,7 +64,7 @@
 public class AttackStylesPlugin extends Plugin
 {
 	private int attackStyleVarbit = -1;
-	private int equippedWeaponTypeVarbit = -1;
+	private int equippedWeaponTypeVarbit = 0; //The player is briefly considered unarmed during login
```

Comment:
What is the purpose of this?

Reply:
The purpose was to avoid an unnecessary weapon switch and thereby a call to processWidgets on login. Should i revert it?

Reply:
This probably isn't always 0 on login? If you change accounts etc it would be whatever it was on the last account

Reply:
The varbit is always set to 0 on login, then updated if you have something equipped. But yea, since i never reset these values when the player logs out they will be kept at their current values and the call will processWidget call will happen twice anyway.

I guess i can reset it on login screen along with widgetHasBeenLoaded to mitigate this.